syntax="proto3";

service HashImage {
  rpc GetHash(ImageRequest) returns (Reply);
}

message ImageRequest {
  bytes image=1;
}

message Reply {
  string hash=1;
}